cognite_toolkit/modules/cdf_data_pipeline_asset_valhall/datasets/dataset.yaml

```diff
@@ -0,0 +1,20 @@
+external_id: 'ds-asset:{{location_name}}'
+name:  'asset:{{location_name}}'
+description: 'Asset data for {{location_name}}'
+metadata: {
+  consoleSource: { 
+    names: [ '{{source_name}}']
+  },
+  rawTables: [
+    {
+      databaseName: 'src-asset-{{location_name}}-{{source_name}}', 
+      tableName: "assets"
```

Comment:
I see now the benefit of NOT having tableName as a variable IF we can keep the table name across source systems.